--- ngrep-1.42/configure.in.org	2004-01-04 21:03:27.000000000 +0000
+++ ngrep-1.42/configure.in	2004-03-29 15:16:33.000000000 +0000
@@ -354,8 +354,7 @@
 
 pcap_dir=""
 for dir in $PCAP_DIR ; do 
-    if test -d $dir -a -r "$dir/pcap.h" -a \
-                       -r "$dir/net/bpf.h" ; then 
+    if test -d $dir -a -r "$dir/pcap.h"; then 
        if test -n "$pcap_dir" -a "$pcap_dir" != "$dir"; then
           echo 
           echo; echo more than one set found in:
--- ngrep-1.42/ngrep.c.org	2004-03-29 02:15:59.000000000 +0000
+++ ngrep-1.42/ngrep.c	2004-03-29 15:18:24.000000000 +0000
@@ -55,7 +55,6 @@
 #include <netinet/ip_icmp.h>
 
 #include <pcap.h>
-#include <net/bpf.h>
 
 #include <stdlib.h>
 #include <string.h>
